src/lib/data/stores.js

```diff
@@ -88,7 +89,9 @@ export const scrolls = groupStore(writable, 'title');
 /**the current view/layout mode*/
 export const viewMode = writable('Single Pane');
 /**is audio active in the app*/
-export const audioActive = writable(true);
+setDefaultStorage('audioActive', config.mainFeatures['audio-turn-on-at-startup']);
+export const audioActive = writable(localStorage.audioActive === 'false');
```

Comment:
should be:

```
export const audioActive = writable(localStorage.audioActive === 'true');
```

You can test by refreshing the browser and the state of the audio button will be the same.